.github/workflows/setup-uds.yaml

```diff
@@ -26,15 +26,18 @@ jobs:
 
   setup-uds:
     name: Full test
+    strategy:
+      matrix:
+        version: [v0.16.0, v0.17.0]
```

Comment:
Should/is this tracked with renovate somewhere to keep this matrix up to date with the latest?

Reply:
It is not and I admit I have no idea where to start there. I suppose I'll start digging

Reply:
This might be somewhat similar to what we're looking to do here: https://github.com/defenseunicorns/uds-rke2-image-builder/blob/main/renovate.json#L26-L35

That ends up tracking the version matrix [here](https://github.com/defenseunicorns/uds-rke2-image-builder/blob/main/.github/workflows/on-pr-aws.yaml#L25). I'm not sure it's great about handling "bump this to `n-1`" though - that is probably some more tricky logic 😅. As-is it will just bump all versions to the latest version which defeats the point of the matrix.

Reply:
In hindsight I'm going to punt on this matrix idea and instead just add renovate and keep the version up to date